src/data/full-quiz.ts

```diff
@@ -1,4 +1,15 @@
 const fullQuiz = [
+  {
+    Question:
+      "Which one of these data structure follows the FIFO (First In First Out) method?",
```

Comment:
Data structure should be plural since you are referring to multiple data structures here. 

```suggestion
      "Which one of these data structures follows the FIFO (First In First Out) method?",
```